components/datetime/src/scaffold/dynamic_impls.rs

```diff
@@ -37,6 +37,7 @@ impl DateInputMarkers for DateFieldSet {
     type MonthInput = datetime_marker_helper!(@input/month, yes);
     type DayOfMonthInput = datetime_marker_helper!(@input/day_of_month, yes);
     type DayOfYearInput = datetime_marker_helper!(@input/day_of_year, yes);
+    type RataDieInput = datetime_marker_helper!(@input/rata_die, yes);
```

Comment:
Thought/Suggestion: Since none of the stock patterns have this field, you could disable it here, and enable it only in power user APIs… hmm, maybe a new marker type? Not sure where to put it.

Reply:
does it hurt here, if it's never set?